src/Trace/Trace.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Trace;
+
+use Google\Cloud\Core\Exception\NotFoundException;
+use Google\Cloud\Core\ValidateTrait;
+use Google\Cloud\Trace\Connection\ConnectionInterface;
+
+/**
+ * This plain PHP class represents a Trace resource. For more information see
+ * [TraceResource](https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects.traces#resource-trace)
+ */
+class Trace
```

Comment:
At this point, probably not. The traceId generally won't be known, unless you're getting it from the trace UI or from a listTraces API call in which case you already know. Or, if it's a random traceId, you probably are actually looking for data about it.